modules/virt-importing-rhel-image-boot-source-web.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-importing-rhel-image-boot-source-web_{context}"]
+= Importing a Red Hat Enterprise Linux image as a boot source
+
+You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by specifying the URL address for the image.
+
+.Prerequisites
+* To import a RHEL image via a URL, access to the web server with the operating system image is required. For example: the Red Hat Enterprise Linux web page with images.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+
+. Click the *Templates* tab.
+
+. Identify the RHEL template that you want to use for creating a virtual machine.
+
+. Click *Add source* for the identified template.
+
+. In the *Add boot source to template* dialog, select *Import via URL (creates PVC) from the *Boot source type*.
+
+. Under the *Import via URL (creates PVC)* field, click *RHEL download page* to access the Red Hat Customer Portal. The Download Red Hat Enterprise Linux web page displays with a list of available installers and images.
+
+. Identify the *Red Hat Enterprise Linux KVM guest image* that you want to download. Right-click *Download Now*, and copy the URL for the image.
+
+. In the *Add boot source to template* dialog, paste the copied URL of the guest image into the *Import URL* field, and click *Save and import*.
```

Comment:
s/dialog/window for consistency with ISG and the other modules in this assembly.

Reply:
Substituted for window.